main/src/vtr_navigation/src/graph_map_server.cpp

```diff
@@ -508,6 +508,9 @@ void GraphMapServer::updateVertexType() {
             ->retrieve<tactic::EnvInfo>("env_info",
                                         "vtr_tactic_msgs/msg/EnvInfo");
     vertex.type = env_info_msg->sharedLocked().get().getData().terrain_type;
+    graph->at(VertexId(vertex.id))->SetTerrainType(vertex.type);
```

Comment:
You need it for CLOG because there is an explicit function header defined for unsigned longs. Without it the conversion is implicitly defined.

I have removed